job_executor/adapter/pseudonym_service.py

```diff
@@ -21,7 +21,8 @@ def pseudonymize(idents: List[str], unit_id_type: str, job_id: str) -> dict:
         headers={
             'Content-Type': 'application/json',
             'X-API-Key': PSEUDONYM_SERVICE_API_KEY
-        }
+        },
+        timeout=10
```

Comment:
This is before establishing a connection or when waiting for response? Pseudonymizing can take some time...

Reply:
You are right, it is the time until it receives data. I'll increase this substantially